starter-bots/kotlin/ReadMe.md

```diff
@@ -0,0 +1,23 @@
+# Kotlin Sample Bot
+
+Kotlin is a statically typed programming language that runs on the Java virtual machine. It is 100% interoperable with Java, but was built to be more expressive, concise and safer.
+
+## Learn Kotlin
+
+The official documentation and tutorials can be found https://kotlinlang.org/
+
+
+## Environment Setup
+
+Install the [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and [Maven](https://maven.apache.org/)
+
+## Compilation
+
+The bot can be built using Maven:
+
+`mvn clean package` compiles the bot and packages it to a jar file. 
+
+## Running
+The bot can be run like any other jar. 
+
+For example: `java -jar kotlin-sample-bot.jar`
```

Comment:
Maybe add a bit like this:
```

Mvn package will create a .jar **?** file in the folder called "target". The file will be called {kotlin-compiled-bot-file}.
To run the bot, copy the file {kotlin-compiled-bot-file} to a different location. Then go to the starter-pack and edit the config.json file accordingly.
Then run the "run.bat" file on windows or the "run.sh" file for unix.
```
Im not sure what the filenames, paths or extensions for kotlin are, just check that the names are correct up here